packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -3078,6 +3078,62 @@ void main() {
     expect(tester.widget<Text>(find.text('helper')).style!.color, helperStyle.color);
     expect(tester.widget<Text>(find.text('counter')).style!.color, counterStyle.color);
 
+    TextStyle getLabelStyle() {
```

Comment:
This is great, but could you make this function global since it is used more than once.